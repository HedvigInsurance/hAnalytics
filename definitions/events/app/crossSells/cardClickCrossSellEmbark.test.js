const validateEvent = require("commons/validateEvent.js");

test("validate", () => {
  const event = validateEvent(__filename, {
    id: "some_id",
    storyName: "name_of_story",
  });
  expect(event).toMatchSnapshot();
});
