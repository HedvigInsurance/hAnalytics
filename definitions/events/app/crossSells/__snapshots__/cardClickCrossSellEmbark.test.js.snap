// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "card_click_cross_sell_embark",
  "variables": Object {
    "id": "some_id",
    "story_name": "name_of_story",
  },
}
`;
